nautobot_circuit_maintenance/urls.py

```diff
@@ -6,6 +6,9 @@
 from .models import CircuitMaintenance, CircuitImpact, Note, NotificationSource
 
 urlpatterns = [
+    # Overview
+    # path("config-compliance/overview/", views.ConfigComplianceOverview.as_view(), name="configcompliance_report"),
+    path("maintenance/overview/", views.CircuitMaintenanceOverview.as_view(), name="circuitmaintenance_overview"),
```

Comment:
Might be good to register this as the "home" page for the plugin? https://nautobot.readthedocs.io/en/stable/plugins/development/#adding-links-to-the-installed-plugins-view

Reply:
Making that update. What does having a home page do for a plugin?

Reply:
It gets linked from the `installed-plugins` list page, i.e.:

![image](https://user-images.githubusercontent.com/5603551/178288271-7d4edd38-7145-493d-8243-c3645525dac2.png)

Reply:
Cool. I had a feeling that was the case. What awesome stuff!